test/ng/httpSpec.js

```diff
@@ -2266,6 +2208,139 @@ describe('$http', function() {
   });
 
 
+  describe('XSRF', function() {
+    var $http;
+    var $httpBackend;
+
+    beforeEach(module(function($httpProvider) {
+      $httpProvider.xsrfWhitelistedOrigins.push('https://whitelisted.example.com/');
```

Comment:
Note that the trailing slash is ignored. Should we have tests for URLs with different paths - to demonstrate that they are ignored?